variables.tf

```diff
@@ -0,0 +1,22 @@
+variable "prefix" {
+  description = "The prefix used for all resources in this environment"
+}
+variable "location" {
+  description = "The Azure location where all resources in this deployment should be created"
+  default     = "uksouth"
+}
+variable "ARM_ACCESS_KEY" {
+    description = "The private key for the tfstate storage container"
+}
```

Comment:
I could be mistaken but you shouldn't need to specify this here; because our service principal has access to the full resource group, it should be able to access the storage container without an explicit access key